build/SharedFxInstaller.targets

```diff
@@ -101,24 +100,27 @@
       <ChangeLogProps>DATE=$([System.DateTime]::UtcNow.ToString(ddd MMM dd yyyy))</ChangeLogProps>
       <ChangeLogProps>$(ChangeLogProps);MAINTAINER_NAME=$(RpmMaintainerName)</ChangeLogProps>
       <ChangeLogProps>$(ChangeLogProps);MAINTAINER_EMAIL=$(RpmMaintainerEmail)</ChangeLogProps>
-      <ChangeLogProps>$(ChangeLogProps);PACKAGE_VERSION=$(RpmVersion)</ChangeLogProps>
+      <ChangeLogProps>$(ChangeLogProps);PACKAGE_VERSION=$(RpmPackageVersion)</ChangeLogProps>
       <ChangeLogProps>$(ChangeLogProps);PACKAGE_REVISION=$(RpmRevision)</ChangeLogProps>
     </PropertyGroup>
 
     <GenerateFileFromTemplate TemplateFile="$(_PackagingDir)changelog.in" OutputPath="$(_WorkRoot)templates/changelog" Properties="$(ChangeLogProps)" />
 
     <!-- Run fpm -->
-    <Exec Command="docker run
+    <!-- Retry added due to fpm/docker race where .w/package_root directory cannot be resolved -->
```

Comment:
Possibly related to this change in build/SharedFxInstaller.targets?
```diff
-<MakeDir Directories="$(_WorkRoot)" />
```

Reply:
I made that change to see if it helped with the flakiness but I guess I can revert that. It should not have any functional effect since the line after we create a subdirectory https://github.com/aspnet/Universe/pull/1140/files#diff-1abc2bfad4e881f4c9ca50362269c055L96